build.sh

```diff
@@ -8,9 +8,11 @@ if [ $EUID -ne 0 ]; then
 	exit 1
 fi
 
-SYSTEM_DESC=GamerOS
-SYSTEM_NAME=gameros
-USERNAME=gamer
+SYSTEM_DESC=${SYSTEM_NAME:-GamerOS}
+SYSTEM_NAME=${SYSTEM_NAME:-gameros}
+USERNAME=${USERNAME:-gamer}
+BUILD_USER=${BUILD_USER:-}
```

Comment:
I'm not sure if this pull request should include adding support for building packages with a specified user, especially since the docker image doesn't seem to use it.

Reply:
@sharkwouter Allowing a different build user is essential to getting it to work in Docker. You can see that it's used in the [Dockerfile](/Dockerfile) defined as an environment variable. This is done because pikaur spawns package builds using `systemd-run` to build packages as non-root, but the container does not have systemd running as its init process.